icons/arrow-down.svg

```diff
@@ -1,5 +1 @@
-<svg width="24px" height="24px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round">
-        <polyline stroke="#000000" stroke-width="2" points="5 9 12 16 19 9"></polyline>
-    </g>
-</svg>
\ No newline at end of file
+<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><path d="M7.25 0v13.13L1.36 7.24.3 8.3 8 16l7.7-7.7-1.06-1.06-5.89 5.89V0h-1.5Z" fill="currentColor"/></svg>
```

Comment:
Potentially something like this? https://css-tricks.com/svg-use-with-external-reference-take-2/#aa-as-a-reminder-it-looks-like-this

```suggestion
<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><symbol id="arrow-down"><path d="M7.25 0v13.13L1.36 7.24.3 8.3 8 16l7.7-7.7-1.06-1.06-5.89 5.89V0h-1.5Z" fill="currentColor"/></symbol></svg>
```